spec/spec_helper.rb

```diff
@@ -1,4 +1,5 @@
 require "bundler"
+require 'berkshelf'
```

Comment:
You are correct, this is no longer needed. At one point I was checking for the existence of the constant `Berkshelf` Eg:

````ruby
      def bundler_with_clean_env(&block)
        if defined?(Bundler)
          Bundler.with_clean_env(&block)
        else
          yield
        end
      end
````

But this ended up not being needed. I should be able to remove this as a development/test dependency